package.json

```diff
@@ -82,6 +82,7 @@
     "@edx/typescript-config": "^1.0.1",
     "@testing-library/jest-dom": "5.17.0",
     "@testing-library/react": "12.1.5",
+    "@testing-library/react-hooks": "^8.0.1",
```

Comment:
I'm adding this library to test the hooks. It's the recommended library from `@terting-library/react`. It was incorporated in `testing-library/react` from versions `>=13`, but it was only compatible with `react` version `>=18` (when are at `17`)